src/wp-content/themes/twentyfourteen/functions.php

```diff
@@ -371,7 +371,19 @@ function twentyfourteen_scripts() {
 		);
 	}
 
-	wp_enqueue_script( 'twentyfourteen-script', get_template_directory_uri() . '/js/functions.js', array( 'jquery' ), '20210122', true );
+	wp_enqueue_script( 'twentyfourteen-script', get_template_directory_uri() . '/js/functions.js', array( 'jquery' ), '2023xxxx', true );
+
+	/*
+	 * Skip-link fix is no longer included in the 'twentyfourteen-script' file.
+	 * If enqueued, the minified inline script targets only Internet Explorer.
+	 * Most of the script comes from Twenty Nineteen, but this repositions the
+	 * window on jump-to-anchor to account for header height.
+	 */
+	wp_register_script( 'twentyfourteen-skip-link-focus-fix', false, array(), false, true );
+	wp_add_inline_script(
+		'twentyfourteen-skip-link-focus-fix',
+		'/(trident|msie)/i.test(navigator.userAgent)&&document.getElementById&&window.addEventListener&&window.addEventListener("hashchange",function(){var t,e=location.hash.substring(1);/^[A-z0-9_-]+$/.test(e)&&(t=document.getElementById(e))&&(/^(?:a|select|input|button|textarea)$/i.test(t.tagName)||(t.tabIndex=-1),t.focus(),window.scrollBy(0,-80))},!1);'
+	);
```

Comment:
I wonder if this is really worthwhile (in this theme and others). Given that no theme/plugin would currently be aware of this script handle, they would have to manually add:

```php
wp_enqueue_script( 'twentyfourteen-skip-link-focus-fix' );
```

And if they are doing that, then they might as well include this whole snippet. 

If we anticipate that only a tiny fraction of users would ever do this, I suggest we remove this otherwise dead code from the themes, and just publish it as part of the dev note.

Reply:
I'll remove these from Twenty Thirteen and Twenty Fourteen. Then I'll test them in my sample child themes.